portfolio/src/main/webapp/chart.html

```diff
@@ -5,8 +5,9 @@
   <script src="https://www.gstatic.com/charts/loader.js"></script>
   <script src="script.js"></script>
 </head>
-<body onload="loadChartsApi()">
+<body onload="loadChartsApi();">
   <h1>Google Virtual Zoo Population</h1>
-  <div id="chart-container"></div>
+  <div id="chart-div"></div>
+  <div id="linechart_material"></div>
```

Comment:
```suggestion
  <div id="linechart-material"></div>
```

https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Reply:
Addressed in "modified id naming scheme to be in compliance with 4.1.10", thanks for linking the resource alongside your comment.